### DISCUSSION TITLE
BloatynosyAI 2.1 Out of Preview!

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#141] [Announcements]
You might have noticed that Bloatynosy [has evolved into BloatynosyAI](https://github.com/builtbybel/BloatynosyAI/releases/tag/2.0). This is due to the current Windows 11 environment and the ongoing AI integration by Microsoft into the operating system.
 
**Last week, the preview of BloatynosyAI was released, and today, we're already able to leave the preview channel.** 
In this final version, the UI along with its core has been further optimized. To make navigation easier, I've aligned the UI with the Windows 11 Start menu. However, most of the action takes place in the intelligent chatbox, similar to what we see in the Windows Search or @Microsoft Copilot. Additionally, support for [Albacore's ViVe Tool](https://github.com/thebookisclosed/ViVe/releases) has been integrated into this version. Popular feature configurations will also be included in BloatynosyAI. Oh, and if you still don't like the name, feel free to call the app CoMachina. Perhaps the app will be renamed again in the future. I'll conduct another survey soon.

![bYT9WUmw2c](https://github.com/builtbybel/BloatynosyAI/assets/57478606/09ffa047-1c12-4e8b-8f90-1f8051d88ef5)

**How does it work?**
The three central modules (Privacy, Bloatware Uninstaller, and App Installer) that we know from Bloatynosy are presented directly on the homepage. Plugins can be accessed, for example, via the keyword "Plugin", "#AI" or "Copilot" to remove AI in Windows 11/10. Additionally, many other Windows functions can be accessed and controlled through the search bar. Just give it a try with some queries or keywords.

_**Just FYI! The app still rocks even if you don't have MSEdge installed! (Though I can't fathom why anyone would ditch the browser entirely). In the rare event the app doesn't kick off, just snag the WebView2 Evergreen Bootstrapper to get things rolling (download it here: https://developer.microsoft.com/en-us/microsoft-edge/webview2?form=MA13LH)**_

**Try out the new version and provide feedback in the [discussions](https://github.com/builtbybel/BloatynosyAI/discussions/140).**
(https://github.com/builtbybel/WhyNotWinAI)_

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/BloatynosyAI/releases/tag/2.1'>BloatynosyAI 2.1 Out of Preview!</a>.</em>

**Comment by FrankRay78** [comment]
![image](https://github.com/builtbybel/BloatynosyAI/assets/52075808/594e1cf0-669c-4ccd-b4da-2a6249c792d1)

Are people actually uninstalling Terminal and Notepad?

**Reply by Belim** [reply]
You're right, at least two of those apps I actually like . Tweaked the algorithm again to make sure the vital ones don't get stuck on the crapware wall 🐱

**Comment by thealexparadox** [comment]
Filename says BoatyNosyAI.zip (no "l"). Was that a typo?

**Reply by Belim** [reply]
Yup, just a typo, thanks for the heads up 🐱
